corehq/apps/reports/templates/reports/reportdata/case_data.html

```diff
@@ -94,13 +83,13 @@ <h4 class="panel-title">
                                 </a>
                               </h4>
                             </div>
-                            <div class="panel-collapse collapse in" id="{{ table.name }}">
-                              <div class="panel-body">
-                                {% include "reports/partials/case_property_table.html" with rows=table.rows %}
-                              </div>
+                          {% endif %}
+                          <div class="panel-collapse collapse in" id="{{ table.name }}">
```

Comment:
will this cause any issues if `table.name` is `None`?  I guess in that case it's not a collapsible target anyways, right?

Reply:
actually should this one and the one on line 80 be `|slugify`ed?
```suggestion
                          <div class="panel-collapse collapse in" id="{{ table.name|slugify }}">
```
It might also be prudent to put a prefix on the table name HTML ids just to ensure they don't conflict with any other IDs on the page.  This may be overly cautious, but there's also an outside possibility that multiple slugified table names conflict.  To prevent that, you could use the loop index instead as a unique identifier:
```suggestion
                          <div class="panel-collapse collapse in" id="property-table-{{ forloop.counter }}">
```